doc/release/README.chplenv

```diff
@@ -238,14 +238,17 @@ Optional Settings
    options are:
 
    CHPL_TASKS:
-        qthreads       : run tasks using Sandia's Qthreads package
-        fifo           : run tasks to completion, in FIFO order
-        massivethreads : run tasks using U Tokyo's MassiveThreads package
+        qthreads       : use Sandia's Qthreads package
+        fifo           : use POSIX threads
+        massivethreads : use U Tokyo's MassiveThreads package
+        muxed          : use Cray-specific lightweight threading
```

Comment:
Ah ok. I think it's fine as is then. Anybody wanting to know more will quickly learn it's only available for the pre-built module by reading README.cray